src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1309,8 +1309,13 @@ private async Task ValidateUnusedIdsAsync(HashSet<string> unusedIds, string data
 
         public static unsafe bool IsBase64String(string base64)
         {
-            int base64Size = (int)Math.Ceiling((double)base64.Length / 3) * 4;
-            Span<byte> bytes = stackalloc byte[base64Size];
+            // base64string length has to be a product of 4
+            var paddingLen = base64.Length%4;
+            for (int i = 0; i < paddingLen; i++)
+                base64 += "=";
+            
+            int base64Size = base64.Length;
+            var bytes = new byte[base64Size];
```

Comment:
Note that you moved from stackalloc to a new array, intentional?
Also, what is the maximum size expected here?
Also, why do we even _do_ that for?

Reply:
Done